src/modules/core/components/ActionsList/ActionsListItem.tsx

```diff
@@ -194,6 +194,8 @@ const ActionsListItem = ({
     motionState === MotionState.Passed ||
     motionState === MotionState.Failed ||
     motionState === MotionState.FailedNoFinalizable;
+
+  const stopPropagation = (event) => event.stopPropagation();
```

Comment:
Why did you make this an external, named, method rather then making it an anonymous defined at render time ?

_(It's good what you did, but I still want to know what was your rationale for it)_

Reply:
2 reasons..... easy!!
- Improved performance: Anonymous functions are not persistent, so TypeScript has to create them again and again on every re-render. Verses creating the named function only once. 
- I checked the codebase for similar situations, to see how you deal with such situations..... and followed your example. ;)